cg/meta/orders/api.py

```diff
@@ -93,8 +91,6 @@ def _create_new_ticket_message(self, order: dict, ticket: dict, project: str) ->
             message = self._add_sample_comment_to_message(message, sample)
 
         message += NEW_LINE
-        message = self._add_project_to_message(message, project)
-        message = self._add_data_delivery_to_message(order, message)
```

Comment:
Is this no longer relevant?

Reply:
I don't know really. I started presenting it because I thought we had the value at hand. Nobody asked for it. It turned out out not to be so readily available so I thought we'll just remove it for now and then add it properly if someone asks for it.